lib/iex/lib/iex/h_erlang_stub.ex

```diff
@@ -0,0 +1,57 @@
+defmodule IEx.H_Erlang_Stub do
+  @moduledoc """
+  Stub module to suggest possible addons for Erlang documentation.
+
+  Currently only returns {:unknown, doc_list }
+  with a suggestive error message.
+  """
+
+  @suggestive "and currently there is no helper installed to provide Erlang documentation"
+
+  def documentation(module) do
+    case is_erlang?(module) do
+      true -> get_doc(module)
+      _    -> {:unknown, [{inspect(module), ""}]}
+    end
+  end
+
+  def documentation(module, function) do
+    case is_erlang?(module) do
+      true -> get_doc(module, function)
+      _    -> {:unknown, [{inspect(module), ""}]}
+    end
+  end
+
+  def documentation(module, function,arity) do
+    case is_erlang?(module) do
+      true -> get_doc(module, function,arity)
+      _    -> {:unknown, [{inspect(module), ""}]}
+    end
+  end
+
+  def is_erlang?(module) do
+    case is_elixir?(module) do
```

Comment:
Does this apply to private functions as well? Cause that should not be a public function. I'll fix it either way. Just curious.